src/components/Chip/ChipWithSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {
+  InputHTMLAttributes,
+  OptionHTMLAttributes,
+  useState,
+} from 'react'
+import styled from '@emotion/styled'
+import { ChevronDown } from '../../icons'
+import { Chip, ChipProps } from './Chip'
+
+const Content = styled.span`
+  select {
+    position: absolute;
+    opacity: 0;
+    inset: 0;
+    cursor: pointer;
+  }
+`
```

Comment:
Why not style the `select` as styled component?

Reply:
Can you explain this? Why doesn't it forward the props, and what's not working then?

Reply:
To be able to create a styled component from another component you have to pass the props to the wrapper of the child component.

So for example:

```js
const Parent = styled(Child)`
  background: red;
`

const Child = () => (
  <div>Hello My Child</div>
)
// ^^^^ This one is not red
```

```js
const Parent = styled(Child)`
  background: green;
`

const Child = (props) => (
  <div {...props}>Hello My Child</div>
)
// ^^^^ This one is green
```

Reply:
Sweet! Done!